rhine-terminal/rhine-terminal.cabal

```diff
@@ -0,0 +1,72 @@
+-- Initial rhine-gloss.cabal generated by cabal init.  For further
+-- documentation, see http://haskell.org/cabal/users-guide/
+
+name:                rhine-terminal
+version:             0.8.0
+synopsis:            Terminal backend for Rhine
+description:
+  This package provides an example of a `terminal` based program using rhine.
+license:             BSD3
+license-file:        LICENSE
+author:              Manuel Bärenz, Jun Matsushita
+maintainer:          programming@manuelbaerenz.de, jun@iilab.org
+-- copyright:
+category:            FRP
+build-type:          Simple
+extra-source-files:  ChangeLog.md
+extra-doc-files:     README.md
+cabal-version:       1.18
+
+source-repository head
+  type:     git
+  location: git@github.com:turion/rhine.git
+
+source-repository this
+  type:     git
+  location: git@github.com:turion/rhine.git
+  tag:      v0.8.0.0
```

Comment:
```suggestion
  tag:      v0.8.0.1
```

Reply:
Should I update all tags in `source-repository` and versions in all rhine packages?

Reply:
Ah sorry, no, I still have the release branch for 0.8.0.1 locally, let me merge that first and then we can rebase.

Reply:
Sounds good 👍

Reply:
Ok I'll stand by on this one.